src/controllers/userController.js

```diff
@@ -168,6 +177,31 @@ const getCurrentUserWithPurchasedAlbums = async (req, res) => {
   */
 };
 
+const updateProfileImage = async (req, res) => {
+  console.log('Updating profile image.');
+  const firebaseStorage = getStorage(firebaseApp);
+  const userId = req.user.userId;
+  if (!req.files || Object.keys(req.files).length === 0) {
+    throw new CustomError.BadRequestError('No files were uploaded.');
+  }
+  const profile = req.files.profile;
+  if (!profile || !profile.data || !profile.mimetype) {
+    throw new CustomError.BadRequestError('File is not an image.');
+  }
+  const imageRef = ref(firebaseStorage, `images/${userId}_profile_pic`);
+  await uploadBytes(imageRef, profile.data, {
+    contentType: profile.mimetype,
+  });
+  const imageUrl = await getDownloadURL(imageRef);
+  const updatedUser = await User.findByIdAndUpdate(
```

Comment:
We'll get a lint warning here about `updatedUser` variable not being used. We can either make the function call without storing the result in a variable; or we can prefix the variable with an underscore, which lets ESLint know that we are intentionally not using this variable later in the function (based on this ESLint configuration: [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/7c9b523158abe7f8e952de14b3bbdca95d4584cf/.eslintrc.json#L8](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/7c9b523158abe7f8e952de14b3bbdca95d4584cf/.eslintrc.json#L8)

```suggestion
    await User.findByIdAndUpdate(
```

or  
 

```suggestion
  const _updatedUser = await User.findByIdAndUpdate(
```